crypto/keys/pem_signer_factory.go

```diff
@@ -21,17 +21,32 @@ import (
 	"crypto/rsa"
 	"crypto/x509"
 	"fmt"
+	"sync"
 
 	"github.com/golang/protobuf/ptypes"
 	"github.com/golang/protobuf/ptypes/any"
 	"github.com/google/trillian"
 	"github.com/google/trillian/crypto/keyspb"
 )
 
+var (
+	pkcs11Module string
```

Comment:
Derp yup never mind, I got myself slightly confused by trying to blindly set it after adding the SignerFactory to the registry instead of before.